ifigon/MealMatchup
# Feature delivery link

# Changes
- Add button to Assign Volunteers page to automatically copy the link to send to volunteers

# Testing
- Go to Assign Volunteers page
- assign volunteers
- click "Copy link" button, assert that text changes to indicate that the link is copied
- paste into url and confirm that link was copied correctly

# Screeshots
### Display before copying
<img width="1425" alt="screen shot 2018-05-22 at 16 16 23" src="https://user-images.githubusercontent.com/9148542/40395139-8be0906c-5ddb-11e8-813c-e3976ed08de3.png">

### Display after copying
<img width="1470" alt="screen shot 2018-05-22 at 16 16 28" src="https://user-images.githubusercontent.com/9148542/40395141-8bf72174-5ddb-11e8-9d30-4002f4ee45c2.png">

### Display on mouseover after copying
<img width="1407" alt="screen shot 2018-05-22 at 16 16 31" src="https://user-images.githubusercontent.com/9148542/40395142-8c1613c2-5ddb-11e8-896f-74a07d3685f2.png">
